### DISCUSSION TITLE
there is a latest version for minecraft 1.7.10/1.7.2?

### DISCUSSION THREAD

**Discussion by FINK89i** [cc-tweaked/CC-Tweaked#2033] [Q&A]
hello i was thinking to download cc-tweaked for minecraft 1.7.10 or 1.7.2 but the only version i could find where from the original computer craft end i dont think it will work like whit programs made on the new versions of cc-tweaked also i may miss new bug fixes or added features so i was wondering if is possible a latest version for forge minecraft 1.7.2 end 1.7.10

**Comment by SquidDev** [comment]
Minecraft 1.7.10 is 10+ years old, and so I'm afraid is unlikely to see support.
